website/content/labs/lab8.md

```diff
@@ -40,7 +42,9 @@ Once the fork has completed, you can clone it and begin your work. Remember to d
 Edit your tasks.txt file and include the URL of your forked cs220-redis-example repository on GitHub.
 
 {{% warning %}}
-Careful! Remember that your tasks.txt is in your personal repository on GitLab (with all your other lab files). You will only be using that repository to edit your tasks.txt file. The rest of your work will happen on the forked cs220-redis-example repository you just created.
+{{% md %}}
+Careful! Remember that your `tasks.txt` is in your personal repository on GitLab (with all your other lab files). You will only be using that repository to edit your `tasks.txt` file. The rest of your work will happen on the forked cs220-redis-example repository you just created.
+{{% /md %}}
 {{% /warning %}}
 
 ## Task 1: Running Docker in the CS VM [Optional]
```

Comment:
Can we link them to the headless guide in this section?

Reply:
I've added a link.